### DISCUSSION TITLE
Weekly Sync 4th April 2022

### DISCUSSION THREAD

**Discussion by matdehaast** [interledger/rafiki#300] [General]
## Updates 

- Brandon
  - added change to `ilp-pay` to autocreate `incoming-payments` 
  - Rafiki needs to still be updated with that change
  - Good amount of work on token introspection, hope to have PR once @sabineschaller `incoming-payments` PR is finished
  - Then need to add the checks of balances based on the grants
- Matt
  - No Rafiki work last week (should be some time to do helm charts this week)
- Nathan
   -  To begin work

## Timelines & Project Scope
[ Main milestones](https://github.com/interledger/rafiki/milestones)

## Review/Spec/TODO
- [ ]

**Comment by cryptopeopleworld** [comment]
hello, on 100% scale - what's the launch status for Rafiki project? thanks
